inject.js

```diff
@@ -347,6 +353,13 @@ function defineVideoController() {
         // insert after parent for correct stacking context
         this.parent.getRootNode().querySelector(".scrim").prepend(fragment);
 
+      //fixes issue #690: site specific vsc-controller stacking.
+      case location.hostname == "www.primevideo.com":
+      case location.hostname == "www.sbs.com.au":
+        // insert before parent to bypass overlay
+        //log("Adding exceptions for ggParent layer stacking.", 5);
+        this.parent.parentElement.parentElement.insertBefore(fragment, this.parent.parentElement.parentElement.firstChild)
+
```

Comment:
should there not be a break; here?(only not sure if tv.apple will then still work because it also doesn't have a break;)